src/omero_zarr/cli.py

```diff
@@ -294,7 +308,21 @@ def _bf_export(self, abs_path: Path, args: argparse.Namespace) -> None:
         if stderr:
             self.ctx.err(stderr.decode("utf-8"))
         if process.returncode == 0:
-            self.ctx.out(f"Image exported to {target.resolve()}")
+            image_source = target / "0"
```

Comment:
It's always `0` despite the series index? Interesting.

Reply:
Yes `--series x,y` is removing the series from the root metadata object and hence reindexing.

Reply:
Unfortunate, it means that you can't then make use of OME-XML if enabled.